Source/WebKit/Shared/API/APIObject.h

```diff
@@ -231,14 +235,18 @@ class Object
 #if DELEGATE_REF_COUNTING_TO_COCOA
 #ifdef __OBJC__
     template<typename T, typename... Args>
-    static void constructInWrapper(NSObject <WKObject> *wrapper, Args&&... args)
+    static void constructInWrapper(id<WKObject> wrapper, Args&&... args)
```

Comment:
> To appease the compiler, also switched the types to "`id`" to avoid a "type not inherited" warning in `WeakObjCPtr.h`.

What (or how many) objects are not inheriting from `NSObject`?  That seems like a bug.  If it's only a handful, we should fix that (perhaps in a follow-up).

Reply:
All of our objects inherit from `NSObject`. However, I think it was the `OBJC_CLASS NSObject` causing the confusion in the compiler.